docs/contributing.md

```diff
@@ -0,0 +1,31 @@
+For any problems with this charm, please [report bugs here](https://github.com/canonical/content-cache-k8s-operator/issues).
+
+The code for this charm can be downloaded as follows:
+
+```
+git clone https://github.com/canonical/content-cache-k8s-operator
+```
+
+To run tests, simply run  `make test`  from within the charm code directory.
```

Comment:
Can you remove the word "simply"?
There are some recommendations about not using the word “simply” (and related) because could be subtly shaming.

Reply:
Also, I don't think we have a `Makefile` any more, so can you update the instructions here as appropriate?